pkg/antctl/raw/multicluster/get/member_token.go

```diff
@@ -0,0 +1,144 @@
+# Get all member tokens in all Namespaces
+  $ antctl mc get membertoken -A
+# Get the specified member token
+  $ antctl mc get membertoken cluster-east-token -n antrea-multicluster
+# Get the specified member token and print the token Secret in YAML format
+  $ antctl mc get membertoken cluster-east-token -n antrea-multicluster -o yaml
+`, "\n")
+
+func (o *tokenOptions) validateAndComplete(cmd *cobra.Command) error {
+	if o.namespace == "" && !o.allNamespaces {
+		return fmt.Errorf("Namespace is required")
+	}
+	if o.allNamespaces {
+		o.namespace = metav1.NamespaceAll
+	}
+
+	var err error
+	if o.k8sClient == nil {
+		o.k8sClient, err = common.NewClient(cmd)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func NewMemberTokenCommand() *cobra.Command {
+	cmdToken = &cobra.Command{
+		Use: "membertoken",
+		Aliases: []string{
+			"membertokens",
+		},
+		Short:   "Print member tokens in a leader cluster",
+		Args:    cobra.MaximumNArgs(1),
+		Example: tokenExamples,
+		RunE:    runEToken,
+	}
+	o := &tokenOptions{}
+	optionsToken = o
+	cmdToken.Flags().StringVarP(&o.namespace, "namespace", "n", "", "Namespace of the token")
+	cmdToken.Flags().StringVarP(&o.outputFormat, "output", "o", "", "Output format. Supported formats: json|yaml")
+	cmdToken.Flags().BoolVarP(&o.allNamespaces, "all-namespaces", "A", false, "Get tokens across all Namespaces")
+	return cmdToken
+}
+
+func runEToken(cmd *cobra.Command, args []string) error {
+	if err := optionsToken.validateAndComplete(cmd); err != nil {
+		return err
+	}
+
+	var memberTokens []corev1.Secret
+	if len(args) > 0 {
+		memberTokenName := args[0]
+		memberToken := corev1.Secret{}
+		err := optionsToken.k8sClient.Get(context.TODO(), types.NamespacedName{
+			Namespace: optionsToken.namespace,
+			Name:      memberTokenName,
+		}, &memberToken)
+		if err != nil {
+			return err
+		}
+
+		memberTokens = append(memberTokens, memberToken)
+	} else {
+		memberTokenList := &corev1.SecretList{}
+		err := optionsToken.k8sClient.List(context.TODO(), memberTokenList, &client.ListOptions{Namespace: optionsToken.namespace})
+		if err != nil {
+			return err
+		}
+		memberTokens = memberTokenList.Items
+	}
+
+	opaqueMemberTokens := []corev1.Secret{}
+	for _, memberToken := range memberTokens {
+		if memberToken.Annotations[common.CreateByAntctlAnnotation] == "true" {
+			opaqueToken := common.ConvertMemberTokenSecret(memberToken)
+			opaqueMemberTokens = append(opaqueMemberTokens, opaqueToken)
+		}
+	}
+
+	if len(opaqueMemberTokens) == 0 {
+		if len(args) > 0 {
+			return fmt.Errorf("Member token %s created by antctl is not found", args[0])
+		}
+		if optionsToken.namespace != "" {
+			fmt.Fprintf(cmd.ErrOrStderr(), "No token found in Namespace %s\n", optionsToken.namespace)
+		} else {
+			fmt.Fprintln(cmd.ErrOrStderr(), "No token found")
+		}
+		return nil
+	}
+
+	err := output(opaqueMemberTokens, false, optionsToken.outputFormat, cmd.OutOrStdout(), membertoken.Transform)
```

Comment:
@luolanzone I still not quite sure about this: seems like it is needed to change `false` to `true` for something like json output? It is weird because `JsonOutput()` does not take the `single bool`

I think what we need is
`opaqueMemberToken = opaqueMemberTokens[0]
output(opaqueMemberToken, true, optionsToken.outputFormat, cmd.OutOrStdout(), membertoken.Transform)`
?